src/libprojectM/Makefile.am

```diff
@@ -14,7 +14,7 @@ AM_CPPFLAGS = \
 lib_LTLIBRARIES = libprojectM.la  # public, possibly-shared library
 
 # link flags
-libprojectM_la_LDFLAGS = $(CG_LDFLAGS) -no-undefined -version-info 2:0:0
+libprojectM_la_LDFLAGS = $(CG_LDFLAGS) -no-undefined -version-info 4:0:2
```

Comment:
How about 3:0:0?
Also update configure.ac

Reply:
In what ways has the API changed since then? As I understand, one should increase the major version only when compatibility with the past has been broken.